src/linux/agent/systemui.js

```diff
@@ -56,6 +56,24 @@ rpc.exports = {
       context.startActivity(launchIntent);
     });
   },
+  broadcastAction: function (packageName, receiverName, action) {
+    return performOnJavaVM(function () {
+
+      if (receiverName === null)
+        throw new Error("Please provide receiver name for '"+ packageName + "'");
```

Comment:
For consistency:
- All strings should be using single-quoted.
- Missing space before `+`. Binary operators should always be surrounded by a space on each side.

Reply:
Oh yes, that would be great!